src/Utils.res

```diff
@@ -1,17 +1,16 @@
-let intEl = n => n |> string_of_int |> React.string
+let intEl = n => React.string(string_of_int(n))
 
-let floatEl = n => n |> Js.Float.toString |> React.string
+let floatEl = n => React.string(Js.Float.toString(n))
 
-let numberEl = n => n |> int_of_float |> string_of_int |> React.string
```

Comment:
`|>` is now fully deprecated and will re-print into an inlined parameter position on its right-hand-side function.

E.g. `[1,2,3] |> Array.map((a) => a +1)` will now re-print into `Array.map((a) => a+1, [1,2,3])`.

The way forward from now on will be using the `->` operator to pipe into the first parameter position and using the new `rescript-core` apis instead.

When designing your own function, always put the collection on the first param position.